.expeditor/verify.pipeline.yml

```diff
@@ -11,45 +11,45 @@ expeditor:
 
 steps:
 
-- label: run-lint-and-specs-ruby-2.4
+- label: run-lint-and-specs-ruby-2.5
   command:
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.4-buster
+        image: ruby:2.5
 
-- label: run-lint-and-specs-ruby-2.5
+- label: run-lint-and-specs-ruby-2.6
   command:
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.5-buster
+        image: ruby:2.6
 
-- label: run-lint-and-specs-ruby-2.6
+- label: run-lint-and-specs-ruby-2.7
   command:
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.6-buster
+        image: ruby:2.7
 
-- label: run-lint-and-specs-ruby-2.7
+- label: run-lint-and-specs-ruby-3.0
   command:
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.7-buster
+        image: ruby:3.0
 
-- label: run-lint-and-specs-ruby-3.0
+- label: run-lint-and-specs-ruby-3.1
   command:
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:3.0-buster
+        image: ruby:3.1
 
 - label: run-specs-windows
```

Comment:
Do we know the version of Ruby for the windows box?

Reply:
Looks like we are not testing this on Ruby 3.1 : https://buildkite.com/chef-oss/chef-mixlib-shellout-main-verify/builds/230#3c137b79-6a73-4f77-aad6-ab518b16303b/97-680 causing https://github.com/chef/ohai/pull/1738 to fail on windows.